test/src/test/java/org/corfudb/benchmark/OptimisticTransactionBenchmark.java

```diff
@@ -0,0 +1,167 @@
+package org.corfudb.benchmark;
+
+import com.google.common.reflect.TypeToken;
+
+import javax.annotation.Nonnull;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+
+import org.openjdk.jmh.annotations.AuxCounters;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.infra.Blackhole;
+
+/**
+ * A benchmark which measures the performance of default (optimistic) transactions.
+ *
+ * Created by zlokhandwala on 5/3/19.
+ */
+public class OptimisticTransactionBenchmark extends AbstractCorfuBenchmark {
+
+    public static class OptimisticBenchmarkState extends CorfuSingleNodeBenchmarkState {
+
+        /**
+         * A runtime to use during the iteration.
+         */
+        CorfuRuntime rt;
+
+        @Param({"10", "100"})
+        int NUM_MAPS;
+
+        @Param({"10", "100"})
+        int NUM_KEYS;
+
+        @Param({"1", "100"})
+        int NUM_OPERATIONS;
+
+        /**
+         * Maps to use.
+         */
+        List<Map<Integer, Integer>> maps;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651477)